src/interact.rs

```diff
@@ -463,6 +512,153 @@ where
     }
 }
 
+#[cfg(not(feature = "async"))]
+#[cfg(feature = "polling")]
+fn interact<P, S, R, W, C>(
+    options: &mut InteractOptions<P, S, R, W, C>,
+    session: &mut Session<P, S>,
+    input: &mut R,
+    output: &mut W,
+) -> Result<(), Error>
+where
+    P: Healthcheck,
+    S: NonBlocking + Read + Write + polling::Source,
+    R: Read + polling::Source,
+    W: Write,
+{
+    use polling::{Event, Poller};
+
+    let mut output_buffer = Vec::new();
+    let options_has_input_checks = !options.input_handlers.is_empty();
+    let mut input_buffer = if options_has_input_checks {
+        Some(Vec::new())
+    } else {
+        None
+    };
+    let mut exited = false;
+
+    // Create a poller and register interest in readability on the socket.
+    let poller = Poller::new()?;
+    poller.add(&input.raw(), Event::readable(0))?;
+    poller.add(&session.get_stream().raw(), Event::readable(1))?;
+
+    println!("HANDLE {}", session.get_stream().raw());
```

Comment:
Debug message should remove.

Reply:
Sorry it took me a while to fix windows built.
And it turned out that on windows we can't do what we do on linux.